pkg/util/validation.go

```diff
@@ -94,3 +95,8 @@ func IsCIdentifier(value string) bool {
 func IsValidPortNum(port int) bool {
 	return 0 < port && port < 65536
 }
+
+// IsValidIP tests that the argument is a valid IPv4 address.
+func IsValidIP(value string) bool {
```

Comment:
To be honest I'd prefer to keep the name as is, because from k8s perspective this verifies whether it's a valid IP (because k8s works only for v4).